activity/scraper.py

```diff
@@ -108,20 +108,23 @@ def get_data(self):
             mon = self.__diff_month(dt)
             if mon < self.CONSTANTS['month_count']:
                 mon = self.CONSTANTS['month_count'] - mon
+                mon = self.CONSTANTS['month_count'] - mon - 1
```

Comment:
Inplace of L110 111 
```mon = 0.5 * self.CONSTANTS['month_count'] - 1```
should do the job ?

Reply:
Damn, sourcetree messed up the staging I think. Fixing.